CMake errors in downstream module

I'm on Ubuntu 2020.10 and with CMake 3.16.3. When building Dumux on top of opm-common and opm-grid, I get CMake errors like
```
CMake Error in examples/liddrivencavity/CMakeLists.txt:
  Found relative path while evaluating include directories of
  "example_ff_liddrivencavity":

    "SYSTEM"
```
for all tests. This happens for opm master and 2020.10 release branch, the 2020.04 release branch is fine.

[console_output.txt](https://github.com/OPM/opm-common/files/5499848/console_output.txt)
[CMakeOutput.log](https://github.com/OPM/opm-common/files/5499850/CMakeOutput.log)
[CMakeError.log](https://github.com/OPM/opm-common/files/5499851/CMakeError.log)

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -76,8 +76,9 @@ macro (config_hook)
 
     # For this project
     add_definitions(-DFMT_HEADER_ONLY)
-    list(APPEND EXTRA_INCLUDES SYSTEM ${PROJECT_SOURCE_DIR}/external/fmtlib/include)
+    include_directories(SYSTEM ${PROJECT_SOURCE_DIR}/external/fmtlib/include)
     include_directories(${EXTRA_INCLUDES} ${PROJECT_BINARY_DIR}/include)
+    list(APPEND EXTRA_INCLUDES ${PROJECT_SOURCE_DIR}/external/fmtlib/include)
 
     # For downstreams
     list(APPEND EXTRA_INCLUDES ${PROJECT_BINARY_DIR}/include)

